src/components/Login.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "./login.css";
+import {NavLink} from "react-router-dom";
+
+function Login(props) {
+
+    return (
+        <>
+            <h1>Login</h1>
+
+            <div id="login">
```

Comment:
Please make use of semantically correct HTML. Don't use div where div is not required or semantically correct.
```suggestion
            <section id="login">
```
Or
```suggestion
            <article id="login">
```